kraft/cmd/lib/add.py

```diff
@@ -72,25 +71,24 @@ def kraft_lib_add(ctx, workdir=None, lib=None):
     metavar="PATH"
 )
 @click.option(
-    '--dump', '-d', 'dumps_local',
+    '--pull/--no-pull', 'pull',
     help='Save libraries into project directory.',
-    is_flag=True,
+    default=True,
 )
 @click.argument('lib', required=False, nargs=-1)
 @click.pass_context
-def cmd_lib_add(ctx, workdir=None, lib=None, dumps_local=False):
+def cmd_lib_add(ctx, workdir=None, lib=None, pull=False):
     """
-    Add a library to the unikraft application project.
+    Remove a library from the unikraft application project.
```

Comment:
The previous comment was ok, I think that you accidentally updated the description of `cmd_lib_add` instead of `cmd_lib_remove`.